plasmapy/formulary/frequencies.py

```diff
@@ -516,3 +518,114 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m**-3) -> u.rad / u.s:
 
 wuh_ = upper_hybrid_frequency
 """Alias to `~plasmapy.formulary.frequencies.upper_hybrid_frequency`."""
+
+
+@validate_quantities(
+    validations_on_return={
+        "units": [u.rad / u.s, u.Hz],
+        "equivalencies": [(u.cy / u.s, u.Hz)],
+    }
+)
+@angular_freq_to_hz
+def Buchsbaum_frequency(
+    B: u.T,
+    n1: u.m**-3,
+    n2: u.m**-3,
+    ion1: ParticleLike,
+    ion2: ParticleLike,
+    Z1=None,
+    Z2=None,
+) -> u.rad / u.s:
+    r"""
+    Calculate the Buchsbaum frequency for a two-ion-species plasma (defaults to units of radians per second.)
+
+    **Aliases:** `wbb_`, `wii_`, `wbi_`
+
+    Parameters
+    ----------
+    B : `~astropy.units.Quantity`
+        The magnetic field magnitude in units convertible to tesla.
+    n1 : `~astropy.units.Quantity`
+        Particle number density of ion species #1 in units convertible to m\ :sup:`-3`.
+    n2 : `~astropy.units.Quantity`
+        Particle number density of ion species #2 in units convertible to m\ :sup:`-3`.
+    ion1 : `~plasmapy.particles.particle_class.Particle`
+        Representation of ion species #1 (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4). If no
+        charge state information is provided, then species #1 is assumed
+        to be singly charged.
+    ion2 : `~plasmapy.particles.particle_class.Particle`
+        Representation of ion species #2 (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4). If no
+        charge state information is provided, then species #2 is assumed
+        to be singly charged.
```

Comment:
Can we, and should we, do a "same as above for the second particle"?

Reply:
Shortened description for ion2